satellite-software/src/BME680Sensor.cpp

```diff
@@ -650,15 +654,20 @@ uint8_t BME680Sensor::handleCommand(uint16_t cmdSequenceID, uint8_t command, uin
         if ((plate_temp < 200)  || (plate_temp > 400)) {
             return CMD_STATUS_BAD_PARAM;
         }
-        PRINT_DEBUG(F("  Gas heat temp = "));
+        PRINT_DEBUG(F("  Heat temp = "));
         PRINT_DEBUG(plate_temp);
         PRINT_DEBUG(F("\n"));
         setGasHeatTemperature(plate_temp);
+    }else if (command == 0x07){
+        PRINT_DEBUG(F("  default "));
+        PRINT_DEBUG(F("\n"));
+        setDefaultValues();
     }
     else { 
         return CMD_STATUS_UNKNOWN_CMD;
     }
 
+    setSensorConfig();
```

Comment:
This method should be called after setting the CRC. The reason for this is that you want to calculate and write the CRC ASAP after up[dating a setting in case the satellite loses power due to the solar panels moving out of view of the sun. If you spend processor time reloading the sensor configuration and you lose power during that, you actually lose every setting because the CRC in the EEPROM doesn't match the settings written to the EEPROM.

Reply:
You are right! I did not think about that!
Change made.